content/en/guides/hosting/data-security/secure-storage-connector.md

```diff
@@ -108,61 +115,70 @@ For details, see [Create a CoreWeave AI Object Storage bucket](https://docs.core
 
-    Replace `<cw-bucket>` with the CoreWeave bucket name and replace `<wb-org-id>` with the W&B organization ID you obtained in step 1.
+1. Configure a bucket policy that grants the required permissions for your W&B deployment to access the bucket and generate [pre-signed URLs]({{< relref "./presigned-urls.md" >}}) that AI workloads in your cloud infrastructure or user browsers utilize to access the bucket. Refer to [Bucket Policy Reference](https://docs.coreweave.com/docs/products/storage/object-storage/reference/bucket-policy) in the CoreWeave documentation.
 
-    ```json
-    {
-      "Version": "2012-10-17",
-      "Statement": [
-      {
-        "Sid": "AllowWandbUser",
-        "Action": [
-          "s3:GetObject*",
-          "s3:GetEncryptionConfiguration",
-          "s3:ListBucket",
-          "s3:ListBucketMultipartUploads",
-          "s3:ListBucketVersions",
-          "s3:AbortMultipartUpload",
-          "s3:DeleteObject",
-          "s3:PutObject",
-          "s3:GetBucketCORS",
-          "s3:GetBucketLocation",
-          "s3:GetBucketVersioning"
-        ],
-        "Effect": "Allow",
-        "Resource": [
-          "arn:aws:s3:::<cw-bucket>/*",
-          "arn:aws:s3:::<cw-bucket>"
-        ],
-        "Principal": {
-          "CW": "arn:aws:iam::wandb:static/wandb-integration-public"
-        },
-        "Condition": {
-          "StringLike": {
-            "wandb:OrgID": [
-              "<wb-org-id>"
-            ]
-          }
-        }
-      },
-      {
-        "Sid": "AllowUsersInOrg",
-        "Action": "s3:*",
-        "Effect": "Allow",
-        "Resource": [
-          "arn:aws:s3:::<cw-bucket>",
-          "arn:aws:s3:::<cw-bucket>/*"
-        ],
-        "Principal": {
-          "CW": "arn:aws:iam::<cw-storage-org-id>:*"
+        ```json
+        {
+          "Version": "2012-10-17",
+          "Statement": [
+          {
+            "Sid": "AllowWandbUser",
+            "Action": [
+              "s3:GetObject*",
+              "s3:GetEncryptionConfiguration",
+              "s3:ListBucket",
+              "s3:ListBucketMultipartUploads",
+              "s3:ListBucketVersions",
+              "s3:AbortMultipartUpload",
+              "s3:DeleteObject",
+              "s3:PutObject",
+              "s3:GetBucketCORS",
+              "s3:GetBucketLocation",
+              "s3:GetBucketVersioning"
+            ],
+            "Effect": "Allow",
+            "Resource": [
+              "arn:aws:s3:::<cw-bucket>/*",
+              "arn:aws:s3:::<cw-bucket>"
+            ],
+            "Principal": {
+              "CW": "arn:aws:iam::wandb:static/wandb-integration-public"
```

Comment:
need to do the same thing as `<wb-org-id>` here as well as for saas there is `wandb-integration-public` and for dedicated its `wandb-integration` for the principal